Makefile.libretro

```diff
@@ -284,7 +284,7 @@ endif
 
 include $(BUILD_DIR)/Makefile.common
 
-OBJS := $(SOURCES_C:.c=.o)
+OBJS := $(SOURCES_C:.c=.o) $(SOURCES_ASM:.S=.o)
```

Comment:
This breaks mingw builds:

src/util/arm-algo.S: Assembler messages:
src/util/arm-algo.S:96: Error: junk at end of line, first unrecognized character
 is `-'
Makefile.libretro:300: recipe for target 'src/util/arm-algo.o' failed